dipy/reconst/dki.py

```diff
@@ -700,8 +800,427 @@ def split_dki_param(dki_params):
 
     return evals, evecs, kt
 
+
+def diffusion_kurtosis_odf(dki_params, sphere, alpha=4):
```

Comment:
Move this function up. It's always a better practice to have functions defined or imported higher up in the file than they are called. It makes the code more readable, because you always know where to look for the import/function definition.